src/components/Counter.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './Counter.scss';
+
+interface State {
+  count: number;
+}
+
+class Counter extends React.Component<{}, State> {
+  state: State = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState((state) => ({ count: state.count + 1 }));
+  };
+
+  add100 = () => {
+    this.setState((state) => ({ count: state.count + 100 }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0) {
+      this.add100();
+    } else {
+      this.addOne();
+    }
```

Comment:
this.addOne() має виконуватись завжди
```suggestion
    }
    this.addOne();
```

Reply:
> this.addOne() має виконуватись завжди

воно змінює поведінку  і додає замість 100 - 101.